README.md

```diff
@@ -42,7 +42,7 @@ Helios is still experimental software. While we hope you try it out, we do not s
 
 `--checkpoint` or `-w` can be used to set a custom weak subjectivity checkpoint. This must be equal to the first beacon block hash of an epoch. Weak subjectivity checkpoints are the root of trust in the system. If this is set to a malicious value, an attacker can cause the client to sync to the wrong chain. Helios sets a default value initially, then caches the most recent finalized block it has seen for later use.
 
-`--network` or `-n` sets the network to sync to. Current valid options are `mainnet` and `goerli`, however users can add custom networks in their configuration files.
+`--network` or `-n` sets the network to sync to. Current valid options are `mainnet` and `holesky`, however users can add custom networks in their configuration files.
```

Comment:
We also support sepolia. Can you add that here too?

Other than that LGTM!

Reply:
Done! also add the related example code

![image](https://github.com/user-attachments/assets/0b38f431-7aa9-4427-a37e-0e09682f801e)